src/functions/keycloak/remove-user-attribute/handler.ts

```diff
@@ -0,0 +1,41 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import { CreateKeycloakApiService } from "src/database/services/keycloak-api-service";
+import { middyfy } from "src/libs/lambda";
+
+/**
+ * Lambda handler to remove a user's attribute
+ * 
+ * @param event API Gateway event
+ * @returns Response message as JSON string
+ */
+const removeUserAttributeHandler: APIGatewayProxyHandler = async (
+  event: APIGatewayProxyEvent
+) => {
+  try {
+    const id = event.pathParameters?.id;
+    const attributeName = event.pathParameters?.attributeName;
+
+    if (!id || !attributeName) {
```

Comment:
I like that the removing and updating of the user attributes has been done in a generic way so it could update any attribute, but I'm also a little concerned it could be too much power when dealing with keycloak!
Perhaps we could add to this check that the attribute name is severaId attribute (whatever we decided to call it).
This could then easily be removed / extended in the future.